packages/data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -3465,14 +3469,14 @@ class Customer {
 
   @ApiProperty({
     required: false,
-    type: Number,
+    type: Decimal,
```

Comment:
why do we need to change this type? why cant we use Number just like we already use for Float?